fe/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -43,11 +43,8 @@
 public abstract class Type {
     private static final Logger LOG = LogManager.getLogger(Type.class);
 
-    // Maximum nesting depth of a type. This limit was determined experimentally by
-    // generating and scanning deeply nested Parquet and Avro files. In those experiments,
-    // we exceeded the stack space in the scanner (which uses recursion for dealing with
-    // nested types) at a nesting depth between 200 and 300 (200 worked, 300 crashed).
-    public static int MAX_NESTING_DEPTH = 100;
+    // Maximum nesting depth of a type.
```

Comment:
Why we just support 2?

Reply:
Just limit the depth of nested types, deeper nested type require a lot of extra development, and I don't think we need to support more than 2 depths, like array<map<array<int>>>......